### PR TITLE
Use the new plugins

### PR DESCRIPTION
**Pull request by lesv**

### CODE REVIEW

**Comment by lesv** [comment]
DO NOT MERGE BY ACCIDENT - must be coordinated with changes to Java Flex Quickstart

**Comment by lesv** [comment]
@tswast PTAL

File: helloworld-servlet/build.gradle [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (tswast) [inline helloworld-servlet/build.gradle]: 2016?

File: helloworld-servlet/README.md [hunk 2]
```diff
@@ -5,13 +5,31 @@
...
-4. Visit http://localhost:8080
+## Setup
 
-## Deploy to AppEngine Managed VMs
+Use either:
+
+* `gcloud init`
+* `gcloud beta auth application-default login`
+
+We support building with [Maven](), [Gradle](https://gradle.org), [Intelij Idea](https://cloud.google.com/tools/intellij/docs/), or [Eclipse]().
```
Comment (tswast) [inline helloworld-servlet/README.md]: Missing links to Maven and Eclipse (Not ready yet?)

**Comment by tswast** [comment]
LGTM. Merge whenever you're ready.

**Comment by lesv** [comment]
Travis passed two back, I don't think we will have a problem, so I'm merging.

**Pull request merged**